.buildbot.sh

```diff
@@ -6,3 +6,7 @@ set -e
 mkdir out && cd out
 cmake -GNinja -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS=OFF -Denable_parallel_mark=ON -Dbuild_tests=ON -Denable_gc_assertions=ON ../
 ninja && ninja test
+
+rm -rf *
```

Comment:
I know I'm paranoid, but I wonder if `cd .. ; rm -rf out ; mkdir out` is a bit safer for some poor future sod who copies and pastes this without thinking?

Reply:
Haha, fair point. Try this: 936f01b640bc0744d8a21c1af473f4623065d365